rust/test_service_psispace/src/lib.rs

```diff
@@ -39,6 +41,40 @@ mod service {
         }
     }
 
+    // TODO: create a macro for unnamed field keys (eg as seen in tuples keys),
+    // so that we can support these types of tables in GQL queries.
+    #[derive(ToKey, InputObject)]
+    pub struct ContentKeyNamed {
+        pub account: AccountNumber,
+        pub path: String,
+    }
```

Comment:
Or should we encourage using structs as keys and not tuples?
@tbfleming @swatanabe thoughts?

Reply:
GraphQL doesn't have a notion of tuples. I think the GraphQL type should always be a struct. We can define `derive(ToKey)` in way that makes the struct usable where a tuple key is expected.

Reply:
If we leave tuples unsupported and encourage structs for keys, then it leaves open the possibility of coming up with a good solution in the future.

Reply:
work TBD for a next PR